ember-headlessui/addon/components/menu/item.ts

```diff
@@ -5,12 +5,26 @@ import { guidFor } from '@ember/object/internals';
 
 import { modifier } from 'ember-modifier';
 
-export default class Item extends Component {
+
+interface MenuItemSignature {
+  Args: {
+    registerItem: (item: Item) => void;
+    unregisterItem: (item: Item) => void;
+    closeMenu: () => void;
+    goToItem: (item: Item) => void;
+    isDisabled?: boolean;
+  };
+  Blocks: {
+    default: [{ isActive: boolean; isDisabled: boolean; Element: Element }];
+  };
+}
+
+export default class Item extends Component<MenuItemSignature> {
   guid = `${guidFor(this)}-tailwindui-menu-item`;
-  element;
+  element?: HTMLElement | null;
```

Comment:
TS complained about this one, probably since the element is registered from the item-element component, and not item directly?